test/moduleView.js

```diff
@@ -0,0 +1,25 @@
+/* global describe,it */
+'use strict';
+
+var assert = require('assert'),
+  nemoFactory = require('nemo-mocha-factory'),
+  plugins = require('./plugins'),
+  nemo = {},
+  setup = {
+    'view': [
+      {
+        'name': 'login',
+        'locator': 'module:nemo-shared-objects/login'
+      }
+    ]
+  };
+
+describe('nemo-view @moduleViewSuite@', function () {
+  nemoFactory({'context': nemo, 'plugins': plugins, 'setup': setup});
+  it('should find the PayPal login email locator', function (done) {
+    var emailLocator = nemo.view.login.emailBy();
+    assert.deepEqual(emailLocator, {'using': 'id', 'value': 'login_email'});
+    done();
+  });
```

Comment:
The tests should no longer be attempting to run webdriver commands on the paypal homepage.. The travis build passed. I'm not sure which tests you mean.